sirbot.yml

```diff
@@ -82,6 +82,9 @@ pythondev:
     seems so professional!\n\n
 
     May your :taco:s be plentiful!
+  moveto:
+    msg: >
+        Looks like conversation is going off topic. Please move the conversation to {}
```

Comment:
Not the same text as in `test.yml`. I guess the correct formatting is the other one